.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2. 
+3. 
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+If applicable, add logs to help explain your problem.
```

Comment:
Should there be a mention how to enable useful traces? I can't seem to find this information from README.md either, so perhaps it should be added there and just have a link to it here.

Reply:
Roger that, fixed it. Please re-check.

Reply:
Better, thanks. IMHO, the information should be in readme.md too, as it might save one from filing a bug in the first place.

Reply:
Yep, makes sense. I'll add that as well.

Reply:
Done!